dcos/dcos-ui
# [WIP] feat(MesosEventManager): implemented event stream based on Mesos V1 API.

Implemented field-wise Mesos event stream using Bacon.js and Immutable.js.

To test:
- Create a custom open cluster using `testing/pull/1795`
- Import `MesosEventManager` and call `MesosEventManager.startConnection()`. This can be done in the MesosStateStore.
- You will see console logs from the stream when you run it
